src/symbols.jl

```diff
@@ -0,0 +1,38 @@
+"""
+    ∅
+
+Empty interval of default flavor and default bounds.
+
+Alias of `emptyinterval()`. Can be typed as `\\emptyset<TAB>`.
+"""
+const ∅ = emptyinterval()
+
+"""
+    ≺(a, b)
+
+Alias of `strictprecedes`. Can be typed as `\\prec<TAB>`.
+"""
+const ≺ = strictprecedes
+
+"""
+    ⪽(a, b)
+
+Alias of `isinterior`. Can be typed as `\\subsetdot<TAB>`.
+"""
+const ⪽ = isinterior
+
+"""
+    ∞
+
+Alias of `Inf`. Can be typed as `\\inf<TAB>`.
+"""
+const ∞ = Inf
```

Comment:
is this really needed if it's not exported (I think it was discussed not to export it)